Aztec/Classes/TextKit/ParagraphProperty/TextList.swift

```diff
@@ -14,14 +14,32 @@ open class TextList: ParagraphProperty {
         case ordered
         case unordered
 
-        func markerText(forItemNumber number: Int) -> String {
+        func markerText(forItemNumber number: Int, indentLevel: Int = 1) -> String {
             switch self {
-            case .ordered:      return "\(number)."
-            case .unordered:    return "\u{2022}"
+            case .ordered:
+                return "\(number)."
+            case .unordered:
+                switch indentLevel {
+                case 1:
+                    return "\u{2022}"
+                case 2:
+                    return "\u{2E30}"
+                default:
+                    return "\u{2B29}"
+                }
             }
         }
     }
 
+    /// List Indent Styles
+    ///
+    public enum IndentStyle: Int {
```

Comment:
Nit: Since this only applies to unordered lists, we should consider making this clearer.